web/src/features/menu/list/index.tsx

```diff
@@ -37,8 +37,8 @@ const ListMenu: React.FC = () => {
   const [indexStates, setIndexStates] = useState<Record<number, number>>({});
   const listRefs = useRef<Array<HTMLDivElement | null>>([]);
 
-  const closeMenu = (ignoreFetch?: boolean, keyPressed?: string) => {
-    if (menu.canClose === false) return;
+  const closeMenu = (ignoreFetch?: boolean, keyPressed?: string, forceClose?: boolean) => {
```

Comment:
Since forceClose isn't a param in Lua (and isn't really needed anyway) you can't you make this the default behavior and not have to add the extra param?

Reply:
Well, if it is default behaviour, then the canClose variable will be useless as it will always force it then when it's not supposed to do it on the usual escape/backspace press

Reply:
Yeah true, good point.